utils/contributors.js

```diff
@@ -59,6 +59,35 @@ const CONTRIBUTERS = [
         `,
         github: "https://github.com/debarati-06",
     },
+    
```

Comment:
Remove this blank line as it can cause merge conflicts

Reply:
okk